src/modules/esl-share/core/esl-share.ts

```diff
@@ -20,36 +17,30 @@ export type ESLShareConfigProviderType = () => Promise<ESLShareConfig>;
  */
 export class ESLShare extends ESLBaseElement {
   public static override is = 'esl-share';
-  protected static _config: Promise<ESLShareConfig>;
-  protected static _popupStore: Map<string, ESLPopup> = new Map<string, ESLPopup>();
 
   /** Register {@link ESLShare} component and dependent {@link ESLShareButton} */
   public static override register(): void {
     ESLShareButton.register();
+    ESLSharePopup.register();
     ESLShareTrigger.register();
```

Comment:
It's funny, but the most critical and painful question appears here.
I am trying to remember if we raised that question in the previous architecture discussions, but we need to make a note somewhere if we do...

As for now, we have ESLTogglable <- ESLPopup <- ESLSharePopoup chain, so if a user uses ESLShare, the whole chain is loaded to his bundle. But what if, as a user, I need just a button/buttons list ...
What strategy do we recommend in that case? (unfortunately ESLPopup chain is not so light to skip it, as a small additional payload)

I cannot say that's a blocker. But if we are going to do something with that in the future, it might be a breaking change.
So we need an argumentation regarding current architecture, or we may have another way (good news, I don't think something significantly changed except the esl-share widget itself)